components/budget/ExpenseBudgetItem.js

```diff
@@ -279,18 +291,20 @@ const ExpenseBudgetItem = ({
             />
           )}
         </Box>
-        {showProcessActions && expense?.permissions && (
-          <ButtonsContainer>
-            <ProcessExpenseButtons
-              host={host}
-              collective={expense.account || collective}
-              expense={expense}
-              permissions={expense.permissions}
-              buttonProps={{ ...DEFAULT_PROCESS_EXPENSE_BTN_PROPS, mx: 1, py: 2 }}
-              onSuccess={onProcess}
-            />
-          </ButtonsContainer>
-        )}
+        {showProcessActions &&
+          expense?.permissions &&
+          ![expenseStatus.REJECTED, expenseStatus.PAID].includes(expense?.status) && (
```

Comment:
Would be great to extract this check `![expenseStatus.REJECTED, expenseStatus.PAID].includes(expense?.status)` to a dedicated variable to avoid duplication. Suggested names: `isPaidOrRejected`, `hasProcessActionsInStatus`